src/lib/QtCore/QSettings.ts

```diff
@@ -2,18 +2,19 @@ import { NativeElement, Component } from '../core/Component';
 import addon from '../utils/addon';
 import { QVariant } from './QVariant';
 
-enum Format {
-    NativeFormat = 0,
-    Registry32Format = 2,
-    Registry64Format = 3,
-    IniFormat = 1,
-    InvalidFormat = 16,
-}
+// The below code in unused and hence commented
```

Comment:
I think these enums need to be exported instead.

You can export these enums and then export them again on the index.ts

Reply:
Other than that it looks amazing!

Reply:
Done \m/

Reply:
yep, those enums will be required for some overloaded QSettings constructors (currently only one constructor is implemented, so those enums are unused, but that's just for now)